types/react-dnd/index.d.ts

```diff
@@ -52,6 +52,9 @@ declare module __ReactDnd {
         backend: Backend
     ): <P>(componentClass: React.ComponentClass<P> | React.StatelessComponent<P>) => ContextComponentClass<P>;
 
+    export class DragDropContextProvider extends React.Component<{ backend: Backend }> {
```

Comment:
can you pull this out into a separate `DragDropContextProviderProps` interface and also add the optional `window` prop?